source/docs/06_guides/03_partner_guide.http.md

```diff
@@ -70,6 +70,20 @@ This request should be authenticated by passing the app identifier and app secre
 `grant_type`
 :    this should be set to `authorization_code`
 
+Example access token request:
+
+```http
+POST https://gocardless.com/oauth/access_token HTTP/1.1
+
+{
+ "client_id": "your app identifier",
```

Comment:
Hey @lewisblackwood great work man!
One small thing- indentation is one space short in this block of code. Cheers